webapp/tests/Unit/Controller/API/GroupControllerTest.php

```diff
@@ -80,8 +80,9 @@ public function testNewAddedGroupPostWithId(): void
         $url = $this->helperGetEndpointURL($this->apiEndpoint);
         $postWithId = $this->newGroupsPostData[0];
         $postWithId['id'] = '1';
-        // CLICS does not allow POST to set the id value
-        $this->verifyApiJsonResponse('POST', $url, 400, 'admin', $postWithId);
+        // CLICS does not allow POST to set the id value. Our API will just ignore the property
+        $returnedObject = $this->verifyApiJsonResponse('POST', $url, 201, 'admin', $postWithId);
+        self::assertNotEquals($returnedObject['id'], $postWithId['id']);
```

Comment:
I'm not sure if I like/understand this,

Why does the behavior change?

Reply:
We used to “accept” and ID and then throw an error. Now we just ignore it.
We could still accept it and throw an error but why should we?

Reply:
Because now you would need to check the returned object if it is the same as what you expect.

Is there a case where you can set the id or is it just never allowed?

Reply:
You can never set it. If you could, we should indeed add it to the DTO